src/Stratis.Bitcoin.Features.BlockStore/Models/Block.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection.Metadata.Ecma335;
+using System.Text;
+using NBitcoin;
+using NBitcoin.RPC;
+
+namespace Stratis.Bitcoin.Features.BlockStore.Models
+{
+    public class Block
```

Comment:
To keep in line with the other model objects in the solution, please call this object BlockModel.
Also, I'm afraid implicitly converting a Block to a Block might lead to confusion, even if they're on different namespaces.